README.md

```diff
@@ -1,42 +1,74 @@
 # BaniDB API
 
-Active work ongoing under dev branch
-=======
+# Active work ongoing under dev branch
+
 [<img height="100" src="http://www.banidb.com/wp-content/uploads/2018/03/full-banidb-logo.png">](http://banidb.com)
 
-[![Email](https://img.shields.io/badge/Email-bod%40khalisfoundation.org-green.svg)](mailto:bod@khalisfoundation.org) 
+[![Email](https://img.shields.io/badge/Email-bod%40khalisfoundation.org-green.svg)](mailto:bod@khalisfoundation.org)
 [![Slack](https://img.shields.io/badge/Slack-join%20the%20conversation-red.svg)](https://khalis.slack.com)
 
 [![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/BJKALklMKao/0.jpg)](https://www.youtube.com/watch?v=BJKALklMKao)
+
+## Contributing
+
+### Prerequisites
+
+1.  [Node](https://nodejs.org/en/download/) (10+)
+    - `nvm` can be used for mac/linux
+2.  [Docker](https://www.docker.com/get-started)
+
+#### Optional
+
+- [MariaDB](https://mariadb.org/download/)
+- [MySQL Workbench](https://dev.mysql.com/downloads/workbench/)
+
+### Installation
+
+- Run `npm ci` in the repository root
+
+### Running
+
+- Run `npm run local` in the repository root
```

Comment:
Should we rename local to dev?

Reply:
We're using the dev script for our pre-prod environment [api](http://api.khajana.org/).

Reply:
I would have liked to use `npm start` but would need @ManjotS to approve and make adjustment to deployment scripts.